pySDC/tests/test_projects/test_DAE/test_problems.py

```diff
@@ -443,17 +443,15 @@ def test_DiscontinuousTestDAE_SDC_detection(M):
     from pySDC.implementations.convergence_controller_classes.basic_restarting import BasicRestartingNonMPI
 
     err_tol = {
-        2: 5.3952e-9,
-        3: 2.6741e-9,
-        4: 1.9163e-8,
-        5: 2.4791e-8,
+        3: 5.97e-13,
```

Comment:
With tolerances so low, I am worried that this test might not work on certain architectures. I believe you are not guaranteed the same machine error below machine precision on different operating systems or CPUs.

Reply:
Mhm, yes. I agree. Maybe the tolerances should be set larger to guarantee that, I'll change that!